modules/redpanda/options.go

```diff
@@ -23,6 +23,9 @@ type options struct {
 	// You must use SCRAM-SHA-256 as algorithm when authenticating on the
 	// Kafka API.
 	ServiceAccounts map[string]string
+
+	// AutoCreateTopicsEnabled is a flag to allow topic auto creation.
+	AutoCreateTopicsEnabled bool
```

Comment:
I think the name is clear enough. Could you propagate this across all references? 🙏 
```suggestion
	AutoCreateTopics bool
```

Reply:
Changed 👍